components/organisms/ContributorProfilePage/contributor-profile-page.tsx

```diff
@@ -159,7 +157,7 @@ const ContributorProfilePage = ({
                   recentContributionCount={recentContributionCount}
                   prVelocity={prVelocity}
                   chart={chart}
-                  openPrs={openPrs}
+                  openPrs={totalPrs}
```

Comment:
refactor (non-blocking): should the `openPrs` prop be renamed to `totalPrs`?

Reply:
Yep, `totalPrs` is better